data_rentgen/db/models/dataset_symlink.py

```diff
@@ -6,7 +6,7 @@
 
 from sqlalchemy import BigInteger, ForeignKey, String, UniqueConstraint
 from sqlalchemy.orm import Mapped, mapped_column, relationship
-from sqlalchemy_utils import ChoiceType
+from sqlalchemy_utils import ChoiceType  # type: ignore[import-untyped]
```

Comment:
Just add this module to ignore in pyproject.toml, instead of copy-pasting this comment all other the code